src/components/pages/CompensationOption/WhatIfIDontAgree.js

```diff
@@ -14,26 +14,35 @@ const WhatIfIDontAgree = ({ resource, resourceCategory }) => {
   return (
     <>
       <Typography variant="h3" className={classes.sectionTitle}>
-        WHAT IF I DON'T AGREE WITH THE JUDGE'S DECISION?
+        WHAT IF I DON'T AGREE WITH THE DECISION?
       </Typography>
 
       {resource.what_if_i_disagree !== null ? (
         resourceCategory === "small-claims-court" ? (
           <Box ml={3}>
-            {resource.what_if_i_disagree.map(step => {
+            {resource.what_if_i_disagree.map((step, idx) => {
               return (
-                <Typography varient="body1" key={step} paragraph={true}>
+                <Typography
+                  varient="body1"
+                  key={step}
+                  paragraph={true}
+                  key={`what-if-i-dont-agree-${idx}`}
+                >
                   <Markdown>{step}</Markdown>
                 </Typography>
               )
             })}
           </Box>
         ) : (
           <ul>
-            {resource.what_if_i_disagree.map(step => {
+            {resource.what_if_i_disagree.map((step, idx) => {
               return (
                 <li key={step}>
-                  <Typography varient="body1" paragraph={true}>
+                  <Typography
+                    varient="body1"
+                    paragraph={true}
+                    key={`what-if-i-dont-agree-${idx}`}
```

Comment:
Do you have to put the key in the li component? (Ignore me if this is not the case)

Reply:
No - but React complains if you don't have it on any iterated component. I don't think it's a big deal here, if it were an interactive component though, it causes weird stuff (React mixes up what event goes where)